joomla/joomla-cms
# [5.1] Joomla Update : extensions check never ends (from #43226)

Original PR #43226 closed by mistake

Summary of Changes
Fix PHP 8.2 deprecated notices during Extensions check while performing Joomla 5.1.0 update

Testing Instructions
Joomla 5.0.3, PHP 8.2 or higher, enable Error reporting to default.
In Joomla Update configuration, set "Potentially incompatible extensions checkbox" to "Show".
Install any non core extension

Update Joomla to 5.1.0-RC shows the Pre-Update Screen.

Actual result BEFORE applying this Pull Request
Extensions check never ends, an error appears in explorer console : uncaught syntaxError..

Expected result AFTER applying this Pull Request
Extensions check works fine

### Link to documentations
Please select:
